### DISCUSSION TITLE
Sticky Sidebar Menu with scroll progression in Next.js

### DISCUSSION THREAD

**Discussion by srinivaspendem** [tailwindlabs/tailwindcss#5159] [Help]
On scroll of the paragraph, sidebar contents should get active. example tailwind docs right sidebar. how to handle it in Next.js

**Comment by alexanderbluhm** [comment]
Since tailwindcss.com is open source you can look up the code.
The scroll update is here: https://github.com/tailwindlabs/tailwindcss.com/blob/master/src/layouts/ContentsLayout.js#L104-L134
and the registration of the headings is here: https://github.com/tailwindlabs/tailwindcss.com/blob/master/src/components/Heading.js#L25-L30

There are also different tutorials, e.g., doing it with an `observer`: https://nickymeuleman.netlify.app/blog/table-of-contents

Nevertheless, this discussion section is used to discuss tailwindcss topics so maybe try the Next.js Discord or other sources :)
